vbos/datasets/models.py

```diff
@@ -57,7 +57,7 @@ class RasterFile(models.Model):
         unique=True,
         validators=[
             FileExtensionValidator(
-                allowed_extensions=["tiff", "tif", "geotiff", "gtiff"]
+                allowed_extensions=["tiff", "tif", "geotiff", "gtiff", "vrt"]
```

Comment:
@willemarcel, could you take a look here and see if this is correct? I thought just adding the `vrt` extension here and running `python manage.py makemigrations` + redeploying should work, but the backend is still not accepting .vrt file uploads. Am I missing a step somewhere?